lms/templates/instructor/instructor_dashboard_2/student_admin.html

```diff
@@ -1,6 +1,13 @@
 <%! from django.utils.translation import ugettext as _ %>
 <%page args="section_data"/>
 
+<script>
```

Comment:
yeah, definitely add that JS file at bottom of the file and store it in static somewhere...